### PR TITLE
Add water filter blog page.

### PR DESCRIPTION
**Pull request by kailamjeter**
With content from this blog post: https://docs.google.com/document/d/10swXBUWZ5ZaeH11p8P1HQImKJCEdRuXDh3xYxLYN_BU/edit?resourcekey=0-_8SY4vAFfEzwmXrGUv1lEA. 

Screenshot: https://screenshot.googleplex.com/3mtLSUsdFoQCfwA
Deployed page: https://kailajeter.leadout-sandbox.blueconduit.com/select-water-filter

Not yet linked anywhere.

### CODE REVIEW

File: client/src/assets/blog/select_water_filter.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import { BlogPost } from '@/assets/blog/blog_post';
+
+export const selectWaterFilter: BlogPost = {
+  title: 'How to Select a Water Filter',
+  image: 'next-steps.jpg',
```
Comment (justinebreuch) [inline client/src/assets/blog/select_water_filter.ts:5]: Could you add a different image bc we don't want duplicates of this photo for the blog home page

File: client/src/router.ts [hunk 2]
```diff
@@ -113,6 +115,18 @@ const routes = [
       image: understandLead.image,
     },
   },
+  {
+    path: SELECT_WATER_FILTER_ROUTE,
```
Comment (justinebreuch) [inline client/src/router.ts:119]: add route attribute to BlogPost and do selectWaterFilter.route here instead of SELECT_WATER_FILTER_ROUTE

**Review by justinebreuch — APPROVED** [review]

File: client/src/assets/blog/select_water_filter.ts (hunk 1 above)
Comment (kailamjeter) [reply client/src/assets/blog/select_water_filter.ts:5]: done

File: client/src/router.ts (hunk 2 above)
Comment (kailamjeter) [reply client/src/router.ts:119]: done

**Pull request merged**